modules/virt-viewing-downward-metrics-tool.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-exposing-downward-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-viewing-downward-metrics-tool_{context}"]
+= Viewing downward metrics by using the vm-dump-metrics tool
+
+You can use the `vm-dump-metrics` tool to view downward metrics.
+
+.Procedure
+
+* To view the downward metrics from the guest virtual machine, edit or create a YAML file as shown in the following example:
```

Comment:
IMHO this may be confusing for the final user. What users should address are two bash commands (`dnf install ...` and `vm-dump-metrics`), instead of a yaml file. I would reword this as:

`To view the downward metrics from the guest virtual machine, address the following commands:`. 
wdyt?